### ISSUE TITLE
_SCHEMA collection poor performance

### ISSUE THREAD

**Issue by MidCry** [ParsePlatform/parse-server#2369]
Hi,

I get a currentOp() performance test from my mongodb (used by self hosted parse server) and saw this periodically huge performance drop queries on "_SCHEMA" collection ...
"_SCHEMA" collection got index on '_id' filed ... migrated from parse app.
Why this happens ?

sample of a currentOp result :
`
{
  "desc": "conn3141",
  "threadId": "140129623627520",
  "connectionId": 3141,
  "client": "127.0.0.1:36959",
  "active": true,
  "opid": 262502607,
  "secs_running": 0,
  "microsecs_running": 120209,
  "op": "query",
  "ns": "jiringland._SCHEMA",
  "query": {
    "find": "_SCHEMA",
    "filter": {}
  },
  "numYields": 0,
  "locks": {
    "Global": "r",
    "Database": "r",
    "Collection": "r"
  },
  "waitingForLock": false,
  "lockStats": {
    "Global": {
      "acquireCount": {
        "r": 2
      }
    },
    "Database": {
      "acquireCount": {
        "r": 1
      }
    },
    "Collection": {
      "acquireCount": {
        "r": 1
      }
    }
  }
}
`

full result is in attached file ...
[CurrentOp.txt](https://github.com/ParsePlatform/parse-server/files/379764/CurrentOp.txt)

**Comment by shSina** [comment]
same issue ...

**Comment by flovilmart** [comment]
That should be solved partially with 2.2.17 with drastically reducing the number of schema requests

**Comment by shSina** [comment]
may info logs after update to version 2.2.17:
I disabled them by commenting [lines](https://github.com/ParsePlatform/parse-server/blob/master/src/Routers/FunctionsRouter.js#L81) and ...
and caused many 5XX and 4XX errors details ,
```js
error: Error generating response. 'bad value in mongoObjectToParseObject' error=bad value in mongoObjectToParseObject
error: Uncaught internal server error. bad value in mongoObjectToParseObject undefined
error: Error generating response. ParseError { code: 141, message: 'UserNotFound' } code=141, message=UserNotFound
bad value in mongoObjectToParseObject
```

**Comment by flovilmart** [comment]
@shSina can you open an issue or even better open a PR with the fix?

**Comment by MidCry** [comment]
what's with all this 5xx and 4xx errors !!!!!!

**Comment by flovilmart** [comment]
That should be fixed with 2.2.17, can you update and report if the performance is still as bad?

**Comment by khirr** [comment]
I have the latest version 2.2.17 and i still have too much queries per second to _SCHEMA, about 80/s, and my normal queries are about 400/s, is it normal?

`_SCHEMA command: find { find: "_SCHEMA", filter: {} } planSummary: COLLSCAN keysExamined:0 docsExamined:30 cursorExhausted:1 keyUpdates:0 writeConflicts:0 numYields:0 nreturned:30 reslen:8520 locks:{ Global: { acquireCount: { r: 2 } }, Database: { acquireCount: { r: 1 } }, Collection: { acquireCount: { r: 1 } } } protocol:op_query 107ms`
